README.md

```diff
@@ -24,15 +24,17 @@ If you are familiar with Docker, build the Docker image (`docker build -t <usern
 
 ## Usage
 
-To upload a file to S3, you must send a POST request with the content of the file and the `Content-Type` header at `/`:
+### Uploading images to **Files**
```

Comment:
The service can be used to upload any kind of file. I guess `### Uploading files to **Files**` is more accurate